daprdocs/content/en/operations/monitoring/metrics/grafana.md

```diff
@@ -142,8 +142,10 @@ First you need to connect Prometheus as a data source to Grafana.
    - Name: `Dapr`
    - HTTP URL: `http://dapr-prom-prometheus-server.dapr-monitoring`
    - Default: On
+   - Skip TLS Verify: On 
+     - Necessary in order to save and test the configuration
 
-   <img src="/images/grafana-prometheus-dapr-server-url.png" alt="Screenshot of the Prometheus Data Source configuration" width=600>
```

Comment:
Is there a reason why you added a new .png image name and not just replaced "grafana-prometheus-dapr-server-url.png" with the new supplied image. And if so I would still call the new image "grafana-prometheus-skip-tls-verify." to stick with the naming convention of the others. Images need to follow naming conventions that relate to the doc they are used in.

Reply:
Nope, no reason - I'll rename the file